doc/source/release_notes.rst

```diff
@@ -1,6 +1,93 @@
 Release History
 ---------------
 
+v1.2.0 (pre-release)
+====================
+
+Features
+********
+
+* On each Signal or Device, attach a Python logger attribute named ``log``
+  with a logger name scoped by module name and the ophyd ``name`` of the
+  parent Device.
+* Signals and Devices now accept ``labels`` argument, a set of labels
+  --- presumed but not (yet) forced to be strings --- which the user can use
+  for grouping and displaying available hardware. The labels are accessible via
+  a new attribute ``_ophyd_labels_``, so name to facilitate duck-typing across
```

Comment:
Was it originally suggested as `__ophyd_labels_` (double underscore)?

Reply:
No, Python reserves those for its own use. When a library adds an interface, it is supposed to use only one underscore, such as IPython's `_repr_html_`.

Reply:
I see, thanks!